style.css

```diff
@@ -0,0 +1,565 @@
+  border-radius: 50%;
+  background-color: rgb(43, 41, 41);
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+}
+
+.social-icon:hover {
+  background-color: rgb(24, 23, 23);
+  cursor: pointer;
+}
+
+.footer-bottom {
+  background-color: black;
+  height: 130px;
+  color: white;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-bottom-text {
+  text-align: center;
+  width: 50%;
+}
+
+.footer-img img {
+  width: 50px;
+  height: auto;
+}
+
+.latest-funding {
+  width: 96vw;
+  margin: auto;
+  margin-top: 50px;
+}
+
+.latest-funding-top {
+  flex-direction: column;
+  color: #f42;
+  padding-left: 10px;
+  padding-bottom: 30px;
+}
+
+.hide-in-phone {
+  display: none;
+}
+
+.latest-funding-top a {
+  color: #f42;
+}
+
+.latest-funding-nav {
+  flex-direction: column;
+}
+
+.latest-funding-nav li {
+  list-style: none;
+}
+
+.card-top-title {
+  width: inherit;
+  margin: auto;
+  height: 50px;
+  background-color: #f42;
+  align-items: center;
+  padding-left: 80px;
+  color: #fff;
+}
+
+.card-top-funding {
+  width: inherit;
+  height: 50px;
+  padding-left: 80px;
+  background-color: rgba(255, 68, 34, 0.08);
+  justify-content: center;
+  flex-direction: column;
+}
+
+.sm-text {
+  font-size: 0.8rem;
+  color: rgb(163, 154, 154);
+}
+
+.card-top-image {
+  width: 50px;
+  position: relative;
+  bottom: 75px;
+  margin-left: 20px;
+}
+
+.card-top-image img {
+  width: 100%;
+}
+
+.funding-card-body {
+  width: inherit;
+  margin: -55px auto 20px;
+  height: 150px;
+  padding-left: 20px;
+  color: #555;
+}
+
+.funding-card-body-top {
+  flex-direction: column;
+  justify-content: space-evenly;
+  height: 70%;
+}
+
+p.sm-text {
+  margin-top: 5px;
+  padding-bottom: 50px;
+}
+
+.hide-funding {
+  display: none;
+}
+
+.sticky {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 99;
+}
+
+/* Media Queries */
+@media screen and (min-width: 770px) {
+  .top {
+    flex-direction: column;
+  }
+
+  .nav-logo,
+  .search-menu-div {
+    display: none;
+  }
+
+  .search-menu-bg {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #fff;
+    width: 100%;
+    height: 35px;
+  }
+
+  .scndlinks {
+    width: 70%;
+    align-items: center;
+    justify-content: space-evenly;
+    list-style: none;
+    font-size: 0.9rem;
+  }
+
+  .blackback {
+    display: flex;
+    justify-content: space-between;
+    background-color: #000;
+    width: 100%;
+    color: rgb(163, 154, 154);
+  }
+
+  .blackback li {
+    display: inline-block;
+    margin: 5px;
+  }
+
+  .latest-funding-nav a {
+    padding: 5px 10px;
+  }
+
+  .blackback li a {
+    color: white;
+  }
+
+  .top-cta {
+    text-align: left;
+    padding: 8px;
+  }
+
+  .highlights {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    gap: 10px;
+    height: 85vh;
+    margin-top: 90px;
+    width: 96vw;
+  }
```

Comment:
Please correct this grid parameter as it is giving a faulty grid as shown in the screenshot. 
![Screenshot from 2021-04-01 00-17-39](https://user-images.githubusercontent.com/64894927/113199163-7b254f80-926f-11eb-9641-1664131ce31e.png)